### PR TITLE
Clear All button

### LINKED ISSUE #36: Clear all button
Add a [clear all]() button to the crafting list to remove all items. A new reducer method could simplify this.

### PR DESCRIPTION
**Pull request by RodEsp**
Adds a button that clears all the selected items.
Fixes #36

### CODE REVIEW

File: src/CraftingPlan.js [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
 
   return (
     <Segment basic>
-      <Header>Crafting:</Header>
+      <Header
+        style={{
+          'max-width': '600px'
+        }}>
+        Crafting:
+        <Button content='Clear All' floated='right' size='mini' compact onClick={e => dispatch({ type: 'REMOVE_ALL' })} />
```
Comment (kyleshrader) [inline src/CraftingPlan.js:15]: what do you think of the basic button? (add basic as a property, like compact)

Comment (RodEsp) [reply src/CraftingPlan.js:15]: I prefer the normal button with the grey background. The styled accordion already has the same look as the `basic` button and both of them together is too much negative white space for my taste.

But honestly, this is your project so you make the call :)
I'm just happy to contribute however I can. No need to consult me on design decisions.

**Comment by RodEsp** [comment]
Btw @kyleshrader how do you want to handle merging these things?
Should the author of each PR merge them once there's an approval?

I was kind of thinking I'd let you merge everything so you have final call on everything, but I'm happy to do whatever you want haha.

**Comment by kyleshrader** [comment]
If there is approval, feel free to merge it. Thank you for making sure though. I think the active button is also more recognizable as a button.

**Comment by kyleshrader** [comment]
Usually, if I don't have blocking comments, just open-ended ones, I'll approve with suggestion

**Pull request merged**